tests/cmd/pmp/pmp.rv64-qemu-virt-supervisor.0.stderr

```diff
@@ -0,0 +1,3 @@
+humility: attached to dump
+thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: TryFromIntError(())', /home/azoghby/env/humility/.cargo/git/checkouts/riscv-0ee1bc3cd203fd65/10d7b85/src/register/pmpaddrx.rs:19:49
```

Comment:
This error seems to be caused by a pmpaddr register set to `0x1fffffffffffffff`, and it being decoded as `napot`.  This is a bug with the `riscv` crate.

Reply:
https://rivosinc.atlassian.net/browse/SW-1013

Reply:
https://github.com/rivosinc/humility/pull/33